Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -561,6 +615,45 @@ def block_stabilizerOrderIso [htGX : IsPretransitive G X] (a : X) :
       apply hB'.smul_eq_of_mem ha'
       exact hBB' <| hgB.symm ▸ (Set.smul_mem_smul_set ha)
 
+/-- The set of blocks for a group action containing a given element is a bounder order -/
+@[to_additive
+"The set of blocks for an additive group action containing a given element is a bounded order"]
+instance Block.boundedOrderOfMem (a : X) :
+    BoundedOrder { B : Set X // a ∈ B ∧ IsBlock G B } where
```

Comment:
How much more do you think you'll use this subtype? Do you think it's worth making it a definition?

Reply:
It is used to reformulate primitive actions in terms of the simplicity of this order.

Reply:
which by the equivalence between that order and the one of subgroups containing the stabilizer of a shows that primitive actions is related to the maximality of that subgroup.

Reply:
(of course, those statements can be formulated with less formalism)

Reply:
Yeah okay, I think you could define `abbrev BlockMem : Type _ := {B : Set X // a ∈ B ∧ IsBlock G B}`